extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -1089,64 +1097,70 @@ private JsonObject getVideoSecondaryInfoRenderer() throws ParsingException {
             final String streamingDataKey,
             final ItagItem.ItagType itagTypeWanted) {
 
+        if (streamingData == null || !streamingData.has(streamingDataKey)) {
+            return new LinkedHashMap<>();
+        }
+
         final Map<String, ItagItem> urlAndItagsFromStreamingDataObject = new LinkedHashMap<>();
```

Comment:
(personal opinion, feel free to ignore)
```suggestion
        final Map<String, ItagItem> urlAndItagsFromStreamingDataObject = new LinkedHashMap<>();
        if (streamingData == null || !streamingData.has(streamingDataKey)) {
            return urlAndItagsFromStreamingDataObject;
        }

```

Reply:
Ignoring it; 
Reason: I don't declare and initialize a variable when it's might not be used...